data/en/structclear.json

```diff
@@ -15,6 +15,20 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/structclear"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/structclear"}
 	},
+	"examples": [
+		{
+			"title": "Script Syntax",
+			"description": "",
+			"code": "profile = {name:'John', instrument:'guitar', occupation:'singer'};\r\nstructClear(profile);\r\nwriteDump(profile);",
+			"result": "An empty struct"
```

Comment:
What about `writeOutput(serializeJSON(profile));` in order to be able to have a testable/runnable result?